Source/Core/CircleGeometry.js

```diff
@@ -73,7 +70,9 @@ define([
             vertexFormat : options.vertexFormat,
             stRotation : options.stRotation
         };
-        this._ellipseGeometry = new EllipseGeometry(ellipseGeometryOptions);
```

Comment:
It actually does matter a lot from a performance standpoint.  While the code will work the same in both cases, it will run slower if you don't define everything int he constructor.